### PR TITLE
feat: adds zksync to network configuration

### PR DESCRIPTION
**Pull request by dutterbutter**
I am making this a **DRAFT PR** as there is an item I require assistance / context from Rainbow team for the following: 

1. What is `balanceCheckerAddress`? I see its a contract, and the comment suggests "Check all the ETH balances at once". I have left this field blank until further direction is provided. 

## What changed (plus any additional context for devs)
- Adds zkSync network configuration

### CODE REVIEW

File: src/networks/zksync.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import config from '@/model/config';
+
+export const getZkSyncNetworkObject = (): NetworkProperties => {
+  return {
+    // wagmi chain data
+    ...zkSync,
+
+    // network related data
+    enabled: config.zkSync_enabled,
+    name: 'ZkSync',
```
Comment (JackHamer09) [inline src/networks/zksync.ts:16]:
```suggestion
    name: 'zkSync',
```

File: src/resources/nfts/simplehash/types.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export enum SimpleHashChain {
   Polygon = 'polygon',
   Zora = 'zora',
   Base = 'base',
+  ZkSync = 'zkSync',
```
Comment (JackHamer09) [inline src/resources/nfts/simplehash/types.ts:12]:
```suggestion
  zkSync = 'zkSync',
```

File: src/networks/types.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ export enum Network {
   bsc = 'bsc',
   zora = 'zora',
   gnosis = 'gnosis',
+  zkSync = 'zkSync',
```
Comment (JackHamer09) [inline src/networks/types.ts:16]: In some places I see network key as lowercase `zksync` and sometimes it's `zkSync`. Should we make all the same?

**Review by JackHamer09 — CHANGES_REQUESTED** [review]

**Comment by dutterbutter** [comment]
@benisgold hey there 👋 

I am hoping to finish up this PR, do you mind giving me some clarity on this variable "balanceCheckerAddress"?

**Pull request closed without merging**